gradoop-examples/src/main/java/org/gradoop/utils/sampling/statistics/WeaklyConnectedComponentsRunner.java

```diff
@@ -0,0 +1,71 @@
+/*
+ * Copyright © 2014 - 2018 Leipzig University (Database Research Group)
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.gradoop.utils.sampling.statistics;
+
+import org.apache.flink.api.common.ProgramDescription;
+import org.apache.flink.api.java.DataSet;
+import org.gradoop.examples.AbstractRunner;
+import org.gradoop.flink.algorithms.gelly.connectedcomponents.WeaklyConnectedComponents;
+import org.gradoop.flink.model.api.epgm.LogicalGraph;
+import org.gradoop.flink.model.impl.functions.tuple.ObjectTo1;
+import org.gradoop.flink.model.impl.operators.sampling.statistics.SamplingEvaluationConstants;
+import org.gradoop.flink.model.impl.operators.statistics.writer.StatisticWriter;
+
+/**
+ * Calls the computation of the weakly connected components for a logical graph. Uses the
+ * Gradoop-Wrapper of Flinks ConnectedComponents-algorithm {@link WeaklyConnectedComponents} with
+ * a fixed maximum number of 50 iterations. Writes the result to a csv-file named
+ * {@value SamplingEvaluationConstants#FILE_WEAKLY_CONNECTED_COMPONENTS} in the output directory,
+ * containing a single line with the number of all weakly connected components, e.g.:
+ * <pre>
+ * BOF
+ * 4
+ * EOF
+ * </pre>
+ */
+public class WeaklyConnectedComponentsRunner extends AbstractRunner implements ProgramDescription {
+
+  /**
+   * Calls the computation of the weakly connected components for the graph.
+   *
+   * <pre>
+   * args[0] - path to graph
+   * args[1] - format of graph (csv, json, indexed)
+   * args[2] - output path
+   * </pre>
+   *
+   * @param args command line arguments
+   * @throws Exception in case of read/write failure
+   */
+  public static void main(String[] args) throws Exception {
+
+    LogicalGraph graph = readLogicalGraph(args[0], args[1]);
+
+    DataSet<Long> wcc = getExecutionEnvironment().fromElements(graph
+      .callForCollection(new WeaklyConnectedComponents(50))
```

Comment:
MaxIteration parameter should not be hard coded. Use another program argument to declare it.